.github/workflows/promote.yaml

```diff
@@ -17,11 +17,13 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - name: Install yq
-        run: apt update && apt install yq -y
+        run: sudo snap install yq
```

Comment:
jfyi yq also exists as a github action

Reply:
I know, but it's one more dependency, and  we can install it easily be one step.